mockk/common/src/test/kotlin/io/mockk/it/MapsTest.kt

```diff
@@ -15,6 +23,8 @@ class Issue35Test {
         val blob: CloudBlockBlob = mockk()
         val metadata: HashMap<String?, String?> = hashMapOf()
         every { blob.metadata } returns metadata
+
+        assertEquals(blob.metadata, metadata)
     }
 
     @Test
```

Comment:
Can you please add a comment here mentioning that this test refers to issue #35?